src/app/components/blogs/blogs.component.html

```diff
@@ -7,18 +7,10 @@
                 <a [href]="blog[0].url" target="_blank">See blog for more details.</a>
             </p>
         </div>
-    
-        <div class="right">
-            <button (click)="view=1">Dismiss</button>
-        </div>
-    </div>
-
-    <div @dismissView class="second-view" *ngIf="view===1">
```

Comment:
Since the blogs component will no longer have the second-view part, we should remove all animations inside the blogs component, as they won't be used.

Reply:
I left the entrance animation and removed the others!